src/components/App.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars, class-methods-use-this,  react/destructuring-assignment */
```

Comment:
You shouldn't be disabling these Lints. They are required to improve your code quality.